internal/authctx/helper.go

```diff
@@ -94,6 +95,32 @@ func ProviderConfigureContext(_ context.Context, d *schema.ResourceData) (interf
 	return setContext(&config)
 }
 
+func ProviderConfigureContextWithDefaultTransport(_ context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
```

Comment:
The comments added is not reflecting here> 
and this function name should be renamed to reflect that it is used for Tests only